docs/content/5.blog/announcing-v2.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: article
+navigation: false
+title: 'Going Full Static'
+description: 'Long awaited features for JAMstack fans has been shipped in v2.13: full static export, improved smart prefetching, integrated crawler, faster re-deploy, built-in web server and new target option for config ⚡️'
+imgUrl: blog/going-full-static/main.jpeg
+imgCredits: Vincent Ledvina
+imgCreditsUrl: https://unsplash.com/@vincentledvina
+date: 2020-06-18
+authors:
+  - name: Sebastien Chopin
+    avatarUrl: https://pbs.twimg.com/profile_images/1042510623962275840/1Iw_Mvud_400x400.jpg
+    link: https://twitter.com/Atinux
+tags:
+  - Nuxt
+  - Static
+  - Crawler
+  - Live Preview
+category: Announcements
+---
+
+# Announcing Nuxt Content v2
+
+2 years after the [release of Content v1](https://github.com/nuxt/content/releases/tag/v1.0.0), we are proud to announce the second version of Nuxt Content with the support of [Nuxt 3](https://v3.nuxtjs.org).
+
+It has been rewritten in TypScript and comes with new features:
+
+- **MDC:** Markdown Components
+- Multi sources (experimental)
+- Locale support (i18n)
+- Navigation generation
+- Live preview edition (coming soon)
```

Comment:
Runtime server support is kinda missing. It is one of the major changes between v1 and v2

Reply:
It was also working in content v1 with `nuxt start` actually

Reply:
Yes but implementation is runtime...